test/parallel/test-http2-noflag.js

```diff
@@ -1,8 +0,0 @@
-// The --expose-http2 flag is not set
```

Comment:
should we inverse this test to ensure it works? Or is that superfluous

Reply:
largely superfluous, but that does remind me that I need to add a test for the env var :-)